### ISSUE TITLE
A11y_AzureDataStudioAAE_Sql Instance 1_Connection Strings_Reflow: Data of the "ADO.NET" edit field is getting lost at 400% zoom

### ISSUE THREAD

**Issue by krrish2021** [microsoft/azuredatastudio#15805]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2; #A11yWCAG2.1;#SH_AzureDataStudio_AAE_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#Zoom;#WCAG1.4.10;#AINotInScope;

### Prerequisite:

- Set resolution to 1280*1024
- Set scaling to 400

### Environment Details:
Application Name: Azure Data Studio -AAE
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure arc and Azure data ClI
2. Connect to Azure Arc Controller
3. Tab till Azure Arc Controller tree and tab to the node present under it
4. Tab to child node "Sqlinstance1" and open manage and tab to "Connection Strings" page
5. Tab to "ADO.NET" edit field and check if data inside it is completely visible or not.

### Actual:
Data of the "ADO.NET" edit field is getting lost at 400% zoom.
similar issue is observed through out the application.

### Expected:
All the data should be visible at 400% zoom or if not visible then tool tip should be provided to it with complete data.

### User Impact:
Low Vision users will face difficulty in getting the data.

**Comment by Charles-Gagnon** [comment]
@krrish2021 I'm closing this as working as intended - contents of edit fields like this can be viewed by using the arrows keys to scroll the content within the field

![zoom](https://user-images.githubusercontent.com/28519865/127541254-31f189f7-b433-4540-8c87-9cf808e56660.gif)

Let us know if this isn't acceptable behavior to meet the guidelines and we can look into other options.

**Comment by krrish2021** [comment]
Closing as not repro

**Comment by Harika1702** [comment]
GitHub Tags:#Closed;
